org.libvips.vipsdisp.json

```diff
@@ -0,0 +1,351 @@
+{
+    "app-id" : "org.libvips.vipsdisp",
```

Comment:
Is it "official" libvips.org project, i.e. will be eventually moved under libvips organization on GitHub? If not, it would be better to use com.github.jcupitt.vipsdisp here (it also implies renaming icons/desktop/appdata files).

Reply:
I plan to move it into the libvips org once it's finally on 1.0, so I'd like to stick with the name.